app/controllers/api/v1/users_controller.rb

```diff
@@ -2,20 +2,19 @@ class Api::V1::UsersController < ApplicationController
 
   def index
     sso_cookie = cookie_data
-    if sso_cookie.blank?
-      return return_bad_request('User')
-    end
-    #get opportunity
+    render json: { status: 'bad request' }.to_json, status: :bad_request and return if sso_cookie.blank?
```

Comment:
You should be able to just `raise BadRequest if sso_coookie.blank?` instead of `render json: { status: 'bad request' }.to_json, status: :bad_request ...`

Reply:
@mwharrison I think that will work when the Pardot PR is merged, but not now.

Reply:
ah, I'll work it into the rebase 👍🏼